### PR TITLE
[TOOLS-2174] multiplatform images

### PR DESCRIPTION
**Pull request by jdogmcsteezy**
Support multiplatform images by incorporating the `TARGETARCH` env var supplied by the docker buildkit.  I provided a script, `multiplatform_build.sh` that will do a multiplatform build and publish the image.  These have to be done in a single step.  You cannot build and then publish.  There are also instructions in there on how to set up your docker environment that should not be in the readme since is not relevant to the public.

The new URLs match the spec from https://aerospike.atlassian.net/browse/TOOLS-2167.  Unless the old artifacts are renamed this file will not support old tools releases.  i.e. Tools 7.3.1 and before.

The SHA is removed to handle both architectures and is grabbed using the appropriate URL.  The alternative is to have both the arm and amd SHA defined in the file.

I also removed some useless files, updated the readme, and updated the license.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,8 +6,10 @@
 
 FROM debian:bullseye-slim 
 
-ENV AEROSPIKE_VERSION 7.3.1
-ENV AEROSPIKE_SHA256 6dd56489f31286ef3124fd900627cb697d07d3661122c8519ca452724f60bb9d
```
Comment (mcoberly2) [inline Dockerfile:10]: As Tools is not part of the Aerospike Official Image, the removal of the SHA from the Dockerfile should not raise an issue with the DockerHub folks.
But @kportertx has sent messages (IRC & Email) requesting further information on whether they will accept a Dockerfile minus the SHA.

File: README.md [hunk 2]
```diff
@@ -32,19 +36,19 @@ Example:
...
+* `asloglatency -h SEED_HOST`
 
-- aql -h SEED_HOST --no-config-file
+* `aql -h SEED_HOST --no-config-file`
 
-- asbackup -h SEED_HOST --no-config-file
+* `asbackup -h SEED_HOST --no-config-file`
 
-- asrestore -h SEED_HOST --no-config-file
+* `asrestore -h SEED_HOST --no-config-file`
```
Comment (mcoberly2) [inline README.md:52]: Could add something for asbench & uda?
* asbench -h SEED_HOST ...
* uda ...

File: Dockerfile (hunk 1 above)
Comment (jdogmcsteezy) [reply Dockerfile:10]: Ah, I did not realize it could be an issue to the DockerHub folks

File: README.md (hunk 2 above)
Comment (jdogmcsteezy) [reply README.md:52]: yes, good catch.

Comment (dwelch-spike) [inline README.md:52]:
```suggestion

* `asbench -h SEED_HOST`

* `uda`

```

@jdogmcsteezy  look good?

**Review by dwelch-spike — APPROVED** [review]
Assuming the new build script works for @mcoberly2 and @pvinh-spike this LGTM.

**Comment by jdogmcsteezy** [comment]
@dwelch-spike,  I will likely delete the build script as it should be incorporated into the release scripts by @pvinh-spike, @mcoberly2

File: README.md [hunk 3]
```diff
@@ -32,19 +36,23 @@ Example:
...
+* `asbackup -h SEED_HOST --no-config-file`
 
-- aql -h SEED_HOST --no-config-file
+* `asrestore -h SEED_HOST --no-config-file`
 
-- asbackup -h SEED_HOST --no-config-file
+* `asbench -h SEED_HOST`
 
-- asrestore -h SEED_HOST --no-config-file
+* `uda`
```
Comment (jdogmcsteezy) [inline README.md:55]:
```suggestion
* `uda` -h SEED_HOST
```

**Pull request merged**